### PR TITLE
rerun: fix rp_visualization TypeError

### PR DESCRIPTION
**Pull request by Willebrew**
Fixed the iteration issue in radar visualization, which was causing a TypeError. Additionally, updated the rerun background color to black for better visibility.

Tested the radar visualization with various replays to ensure no TypeErrors occur. Verified that the background color change does not affect other rerun UI elements.

### CODE REVIEW

File: tools/replay/lib/rp_helpers.py [hunk 1]
```diff
@@ -69,25 +70,24 @@ def plot_lead(rs, lid_overlay):
 
 
 def maybe_update_radar_points(lt, lid_overlay):
-  ar_pts = []
-  if lt is not None:
+  if lt is not None and hasattr(lt, 'trackList'):
```
Comment (adeebshihadeh) [inline tools/replay/lib/rp_helpers.py:73]: this should just be handled in the caller. not sure why it was written like this with the "maybe"

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]

File: tools/replay/lib/rp_helpers.py (hunk 1 above)
Comment (Willebrew) [reply tools/replay/lib/rp_helpers.py:73]: I agree it's quite strange.

Comment (Willebrew) [reply tools/replay/lib/rp_helpers.py]: It's now handled in the caller and it passed all tests.

File: tools/replay/rp_visualization.py [hunk 2]
```diff
@@ -33,7 +33,8 @@ def visualize(addr):
     if sm.recv_frame['radarState']:
       plot_lead(sm['radarState'], lid_overlay)
     liveTracksTime = sm.logMonoTime['liveTracks']
-    maybe_update_radar_points(sm['liveTracks'], lid_overlay)
+    if sm['liveTracks'] is not None and hasattr(sm['liveTracks'], 'trackList'):
```
Comment (adeebshihadeh) [inline tools/replay/rp_visualization.py:36]: this should work
```suggestion
    if sm.updated['liveTracks']:
```

Comment (adeebshihadeh) [reply tools/replay/rp_visualization.py:36]: and with that, can we remove the "maybe_"?

Comment (Willebrew) [reply tools/replay/rp_visualization.py]: Haha agreed, done.

**Pull request merged**